{
  try {
    return errorFile.exists();
  }
 catch (  IOException e) {
    throw new KettleException(e);
  }
}
